Makefile

```diff
@@ -147,6 +147,10 @@ lint-api-fix: $(GOLANGCI_LINT_KAL)
 lint-config: $(GOLANGCI_LINT) ## Verify golangci-lint linter configuration
 	$(GOLANGCI_LINT) config verify
 
+.PHONY: verify-boilerplate
+verify-boilerplate: ## Verify boilerplate text exists in each file
+	 ./hack/verify-boilerplate.sh
+
```

Comment:
lets update this to now call the `verify-all.sh` script (which will in turn take care of calling `verify-boilerplate.sh`)